packages/@o3r/extractors/package.json

```diff
@@ -34,6 +43,30 @@
     },
     "@schematics/angular": {
       "optional": true
+    },
+    "@yarnpkg/cli": {
+      "optional": true
+    },
+    "@yarnpkg/core": {
+      "optional": true
+    },
+    "@yarnpkg/fslib": {
+      "optional": true
+    },
+    "@yarnpkg/plugin-npm": {
+      "optional": true
+    },
+    "@yarnpkg/plugin-pack": {
+      "optional": true
+    },
+    "globby": {
+      "optional": true
+    },
+    "semver": {
+      "optional": true
+    },
+    "type-fest": {
```

Comment:
This package is required to make the typings exported by `@o3r/extractors` correct.
Otherwise, the consumer of the library would get implicit `any` instead of `JsonObject` for example
Given that, I think it's correct to add it as optional peerDependency